gradle/libs.versions.toml

```diff
@@ -1,42 +1,28 @@
 [versions]
 ## SDK Versions
 minSdk = "21"
-targetSdk = "34"
-compileSdk = "34"
+targetSdk = "35"
+compileSdk = "35"
 
 # Dependencies
-kotlin = "1.9.22"
-android-gradle-plugin = "8.2.0"
-coroutines = "1.8.0"
-kotlinx-datetime = "0.5.0"
-ktor = "2.3.9"
+kotlin = "2.1.0"
+android-gradle-plugin = "8.8.0"
 
-compose = "1.5.4"
-composeCompiler = "1.5.8"
+compose = "1.7.6"
+composeCompiler = "1.5.15"
 
 # Sample - Android
-androidx-core = "1.12.0"
-androidx-lifecycle = "2.7.0"
-androidx-activity-compose = "1.8.2"
-android-desugaring = "2.0.4"
-koin = "3.5.3"
+androidx-core = "1.15.0"
+androidx-lifecycle = "2.8.7"
+androidx-activity-compose = "1.10.0"
+android-desugaring = "2.1.4"
+koin = "4.0.0"
 
-sharedlib = "0.1.1"
+sharedlib = "0.1.0"
```

Comment:
you removed sharedLib from the other samples. Is it needed here?

Reply:
Yes because this is the Android project that implements the shared lib, while the other samples were creating the shared lib